pandas/tests/indexing/test_loc.py

```diff
@@ -2072,3 +2072,21 @@ def test_loc_setitem_dt64tz_values(self):
         s2["a"] = expected
         result = s2["a"]
         assert result == expected
+
+    @pytest.mark.parametrize("array_fn", [np.array, pd.array, list, tuple])
```

Comment:
hmm can you add Series as an array_fn as well?

Reply:
I thought it would work...
`series.loc[x] = Series(...)` raises another error: `ValueError: No axis named 1 for object type Series` (on master and on this branch)
With `Series.iloc`, the behaviour is like for the rest, raises on master when same size, corrected on this branch

Reply:
created #38303